nixos/modules/services/web-apps/hledger-web.nix

```diff
@@ -0,0 +1,77 @@
+{ lib, pkgs, config, ... }:
+with lib;                      
+let
+  cfg = config.services.hledger-web;
+in {
+  options.services.hledger-web = {
+
+    enable = mkEnableOption "hledger-web service";
+
+    serveApi = mkEnableOption "Serve only the JSON web API, without the web UI.";
+
+    host = mkOption {
+      type = types.str;
+      default = "127.0.0.1";
+      description = ''
+        Address to listen on.
+      '';
+    };
+
+    port = mkOption {
+      type = types.port;
+      default = 5000;
+      example = "80";
+      description = ''
+        Port to listen on.
+      '';
+    };
+
+    capabilities = mkOption {
+      type = types.commas;
+      default = "view,add";
+      description = ''
+        Enable the view, add, and/or manage capabilities.
+      '';
+    };
+
+    file = mkOption {
+      type = types.path;
+      example = "~/.hledger.journal";
+      description = ''
+        Input journal file.
+      '';
+    };
+    
+    baseUrl = mkOption {
+      type = with types; nullOr str;
+      default = null;
+      example = "hledger.org";
```

Comment:
How about using `example.org` here? It's probably not a good idea to show a host name here that really exists. Furthermore, I am confused because the documentation calls this an "URL" yet what the example shows is not an URL but a host name.

Reply:
changed it to https://example.org